traversability_estimation/include/traversability_estimation/TraversabilityMap.hpp

```diff
@@ -150,6 +150,15 @@ class TraversabilityMap
     */
   void restoreDefaultTraversabilityUnknownRegionsReadAtInit();
 
+  /*!
+   * Checks if given 2D pose (x,y) belongs to the traversability layer of the current map.
+   * @param x X coordinate of the 2D pose to check.
+   * @param y Y coordinate of the 2D pose to check.
+   * @return true if pose belongs to the traversability layer of the current map,
+   *              i.e. if there is a valid traversability value in that position.
+   */
+  bool is2DPoseInTraversabilityMap(double x, double y) const;
```

Comment:
I think we should rename to something like "positionHasValidTraversability" .
We only take a position as input, so we should not name it Pose. I'd be fine with omitting the "2D" since the traversability map lives in 2D space anyway, so that's implied anyway.

Also, this function checks if the cell at a given position has a valid traversability value, right? The name "is...InMap" might also suggest that the position simply lies inside the rectangular base grid_map of the traversability map so I think we should be more clear about that.

Reply:
Alternate name suggestion: "MapHasValidTraversabilityAt"

Reply:
I agree :+1: